lib/Cake/Utility/String.php

```diff
@@ -355,6 +355,23 @@ public static function wrap($text, $options = array()) {
  * @return string Formatted text.
  */
 	public static function wordWrap($text, $width = 72, $break = "\n", $cut = false) {
+		$paragraphs = explode($break, $text);
+		foreach ($paragraphs as &$paragraph) {
+			$paragraph = String::_wordWrap($paragraph, $width, $break, $cut);
```

Comment:
Makes me imagine that you are passing a string reference into the function. It is just cosmetic.